src/Microsoft.Data/DataFrameTable.cs

```diff
@@ -138,6 +138,76 @@ public void RemoveColumn(string columnName)
             }
         }
 
+        private void ResizeByOneAndAppend(BaseColumn column, object value)
+        {
+            long length = column.Length;
+            column.Resize(length + 1);
+            column[length] = value;
+        }
+
+        public void Append(IEnumerable<object> row)
+        {
+            IEnumerator<BaseColumn> columnEnumerator = _columns.GetEnumerator();
+            bool columnMoveNext = columnEnumerator.MoveNext();
+            if (row != null)
```

Comment:
Do we want to throw if the user passed in `null`?

Reply:
I don't think so. It seems like a convenient way to append a row of `null`s to a DataFrame?

Reply:
Seems like this behavior should be documented then, if it is intended as the way to append a row of `null`s to the DataFrame.